fastlane/lib/fastlane/plugins/plugin_manager.rb

```diff
@@ -373,7 +373,7 @@ def store_plugin_reference(gem_name)
       # (a plugin may contain any number of actions)
       version_number = Fastlane::ActionCollector.determine_version(gem_name)
       references = Fastlane.const_get(module_name).all_classes.collect do |path|
-        next unless File.dirname(path).end_with?("/actions") # we only want to match actions
+        next unless File.dirname(path).include?("/actions") # we only want to match actions
```

Comment:
Could this be to "lax" in some way? Right now it is quite strict looking for `..../actions`. Is there any way that `.../actions...` could lead to problems? What values can `path` have?

Reply:
> What values can path have?

They are the values defined by the plugin in `all_classes`. 
And this is the point: in the plugin you can do something like 
```
def self.all_classes
      Dir[File.expand_path('**/{actions,helper}/**/*.rb', File.dirname(__FILE__))]
end
```
and it will load all the actions in `actions` subfolders. 
But they aren't listed when `fastlane` starts because of the `end_with?` limit.

>  Is there any way that .../actions... could lead to problems?

The only bad case I could think about was that if a plugin has `helpers` with a name that starts with "actions", they will pass the filter and get listed together with the other actions. 

Two thought about this:
- the initial `/` in the filter should make sure that only helpers with a name that _starts_ with "actions" could pass the filter. If "actions" is used anywhere else in the helper's name, they will be filtered out.
- the worst case is that a helper is listed in the initial message. This is probably as bad as not having actions listed like now, but it still shouldn't affect the plugin functionalities.

Reply:
I agree, this sounds like a win.